doc/source/2-HOW-TO/CASE2-MODDING-PWNISLAND.rst

```diff
@@ -0,0 +1,13 @@
+Case 1: Modding the video game pwn island
```

Comment:
I like the idea of having some tutorials, but I am not sure this is the most adequate one. Also, we would not want to merge anything incomplete.

If you have some material on this, we would be happy to include a link to an external page in the documentation.

Reply:
I was going to write this article. From far too early! I may need to remove this article from ddisasm.